subsys/bluetooth/services/lls.c

```diff
@@ -0,0 +1,75 @@
+
+/** @file
+ *  @brief Link Loss Service
+ */
+
+#include <zephyr/types.h>
+#include <stddef.h>
+#include <string.h>
+#include <errno.h>
+#include <sys/printk.h>
+#include <sys/byteorder.h>
+#include <zephyr.h>
+
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/hci.h>
+#include <bluetooth/conn.h>
+#include <bluetooth/uuid.h>
+#include <bluetooth/gatt.h>
+
+#include <bluetooth/services/lls.h>
+
+#include <logging/log.h>
+
+LOG_MODULE_REGISTER(bt_nrf_lls, CONFIG_BT_NRF_LLS_LOG_LEVEL);
+
+static volatile uint8_t alert_level = 0;
```

Comment:
Isn't this LLS per-connection ? I mean, shouldn't we have one `alert_level` for each (possible) connection ?